lib/csv_to_popolo.rb

```diff
@@ -3,169 +3,171 @@
+      aliases: %w(
+        party_id faction_id faktion_id bloc_id block_id org_id
+        organization_id organisation_id
+      )
     },
-    honorific_prefix: { 
-      type: 'asis',
+    honorific_prefix: {
+      type: 'asis'
     },
-    honorific_suffix: { 
-      type: 'asis',
+    honorific_suffix: {
+      type: 'asis'
     },
-    id: { 
+    id: {
       # TODO: default
-      type: 'asis',
+      type: 'asis'
     },
-    image: { 
+    image: {
       aliases: %w(img picture photo photograph portrait),
-      type: 'asis',
+      type: 'asis'
     },
-    instagram: { 
-      type: 'link',
+    instagram: {
+      type: 'link'
     },
-    linkedin: { 
-      type: 'link',
+    linkedin: {
+      type: 'link'
     },
-    name: { 
+    name: {
       type: 'asis',
-      aliases: %w(name_en),
+      aliases: %w(name_en)
     },
-    national_identity: { 
-      type: 'asis',
+    national_identity: {
+      type: 'asis'
     },
-    patronymic_name: { 
+    patronymic_name: {
       aliases: %w(patronym patronymic),
-      type: 'asis',
+      type: 'asis'
     },
-    phone: { 
+    phone: {
       aliases: %w(tel telephone),
-      type: 'contact',
-    }, 
-    sort_name: { 
-      type: 'asis',
+      type: 'contact'
     },
-    start_date: { 
-      aliases: %w(start started from since),
+    sort_name: {
+      type: 'asis'
     },
-    summary: { 
-      type: 'asis',
+    start_date: {
+      aliases: %w(start started from since)
     },
-    term: { 
-      aliases: %w(legislative_period),
+    summary: {
+      type: 'asis'
     },
-    twitter: { 
-      type: 'contact',
-    }, 
-    website: { 
-      type: 'link',
-      aliases: %w(homepage href url site),
+    term: {
+      aliases: %w(legislative_period)
     },
-    wikipedia: { 
-      type: 'link',
+    twitter: {
+      type: 'contact'
     },
-    youtube: { 
+    website: {
       type: 'link',
+      aliases: %w(homepage href url site)
+    },
+    wikipedia: {
+      type: 'link'
+    },
+    youtube: {
+      type: 'link'
     },
-    
-    other_name: { },
-  }
 
-  def self.model 
-    @@model 
-  end
+    other_name: {}
+  }
 
   class CSV
+    KEY_MAP = MODEL
+              .select { |_, v| v.key? :aliases }
+              .map { |k, v| v[:aliases].map { |iv| { iv => k } } }
+              .flatten.reduce({}, :update)
 
-    @@key_map = Popolo.model.find_all { |k, v| v.has_key? :aliases }.map { |k, v| 
-      v[:aliases].map { |iv| { iv => k } } 
-    }.flatten.reduce({}, :update)
-
-    @@opts = {
+    OPTS = {
       headers: true,
-      header_converters: lambda { |h| 
+      header_converters: lambda do |h|
         # = HeaderConverters.symbol + remapping
-        hc = h.to_s.encode(::CSV::ConverterEncoding).downcase.gsub(/\s+/, "_").gsub(/\W+/, "")
-        (@@key_map[hc] || hc).to_sym
-      }
+        hc = h.to_s.encode(::CSV::ConverterEncoding).downcase.gsub(/\s+/, '_').gsub(/\W+/, '')
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.